docs/DeveloperGuide.md

```diff
@@ -2,32 +2,143 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+- NUS SC2113 Duke project
+- [NUS SC2113 tp](https://github.com/nus-cs2113-AY2223S2)
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+All commands will be fetched by the getUserCommand() method in the TextUi class. The parseCommand() method in the Parser class will then parse the command and pass it to the respective command classes. The command classes will be responsible for executing the relevant features.
+
+### [Proposed] Search book by title
+
+![](images/Search%20class%20diagram.png)
```

Comment:
Since you are documenting a specific feature, a sequence diagram could be more suitable rather than a class diagram